lib/iris/tests/integration/fast_load/test_fast_load.py

```diff
@@ -583,6 +521,109 @@ def test_missing_combination(self):
+            # It can't combine the scalar and vector time coords.
+            expected = CubeList([CubeList(multi_timepoint_flds).merge_cube(),
+                                 single_timepoint_fld])
+            # NOTE: in this case, we need to sort the results to ensure a
+            # repeatable ordering, because ??somehow?? the random temporary
+            # directory name affects the ordering of the cubes in the result !
+            results = CubeList(sorted(results,
+                                      key=lambda cube: cube.shape))
+
+        self.assertEqual(results, expected)
+
+    def test_FAIL_phenomena_nostash(self):
+        # If we remove the 'STASH' attributes, certain phenomena can still be
+        # successfully encoded+decoded by standard load using LBFC values.
+        # Structured loading gets this wrong, because it does not use LBFC in
+        # characterising phenomena.
+        flds = self.fields(c_t='1122', phn='0101')
+        for fld in flds:
+            del fld.attributes['STASH']
+        file = self.save_fieldcubes(flds)
+        results = iris.load(file)
+        if not self.do_fast_loads:
+            # This is what we'd LIKE to get (what iris.load gives).
+            expected = CubeList(flds).merge()
+        else:
+            # At present, we get a cube incorrectly combined together over all
+            # 4 timepoints, with the same phenomenon for all (!wrong!).
+            # It's a bit tricky to arrange the existing data like that.
+            # Do it by hacking the time values to allow merge, and then fixing
+            # up the time
+            old_t1, old_t2 = (fld.coord('time').points[0]
+                              for fld in (flds[0], flds[2]))
+            for i_fld, fld in enumerate(flds):
+                # Hack the phenomena to all look like the first one.
+                fld.rename('air_temperature')
+                fld.units = 'K'
+                # Hack the time points so the 4 cube can merge into one.
+                fld.coord('time').points = [old_t1 + i_fld]
+            one_cube = CubeList(flds).merge_cube()
+            # Replace time dim with an anonymous dim.
+            co_t_fake = one_cube.coord('time')
+            one_cube.remove_coord(co_t_fake)
+            # Reconstruct + add back the expected auxiliary time coord.
+            co_t_new = AuxCoord([old_t1, old_t1, old_t2, old_t2],
+                                standard_name='time', units=co_t_fake.units)
+            one_cube.add_aux_coord(co_t_new, 0)
+            expected = [one_cube]
+        self.assertEqual(results, expected)
+
+    def test_FAIL_pseudo_levels(self):
+        # Show how pseudo levels are handled.
+        flds = self.fields(c_t='000111222',
+                           pse='123123123')
+        file = self.save_fieldcubes(flds)
+        results = iris.load(file)
+        expected = CubeList(flds).merge()
+
+# NOTE: this problem is now fixed : Structured load gives the same answer.
```

Comment:
following an off line discussion about this I am persuaded to leave this in for now, as this commit will be squished so it will never be in teh history.

the handling of pseudo levels may require more effort, so these reminders may prove useful